modules/core/04-channel/keeper/packet.go

```diff
@@ -471,7 +471,7 @@ func (k Keeper) AcknowledgePacket(
 	commitment := k.GetPacketCommitment(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
 
 	if len(commitment) == 0 {
-		return sdkerrors.Wrapf(types.ErrPacketCommitmentNotFound, "packet with sequence (%d) has been acknowledged, or timed out. In rare cases the packet was never sent or the packet sequence is incorrect", packet.GetSequence())
+		return sdkerrors.Wrapf(types.ErrPacketCommitmentNotFound, "packet with sequence (%d) has been acknowledged, or timed out. In rare cases, the packet referenced was never sent, likely due to the relayer being misconfigured", packet.GetSequence())
```

Comment:
how does this sound?

Reply:
the thing is that it might be another relayer, right?

Reply:
implies the existence of a misconfigured relayer?

Reply:
> the thing is that it might be another relayer, right?

No, I don't think so. The only time I can reasonably see the rare case part occurring is if a relayer specifies the wrong packet sequence. Instead of 100, it puts 1000 or something like that

> implies the existence of a misconfigured relayer?

I suppose, but this error should be returned to the misconfigured relayer. Another misconfigured relayer would not cause this error message for a correct relayer

Reply:
Ah, I see, never mind, all good then.